<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>QtCaesar</class>
 <widget class="QDialog" name="QtCaesar">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>845</width>
    <height>556</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <widget class="QDialogButtonBox" name="buttonBox">
   <property name="geometry">
    <rect>
     <x>470</x>
     <y>490</y>
     <width>341</width>
     <height>32</height>
    </rect>
   </property>
   <property name="orientation">
    <enum>Qt::Horizontal</enum>
   </property>
   <property name="standardButtons">
    <set>QDialogButtonBox::Cancel|QDialogButtonBox::Ok</set>
   </property>
  </widget>
  <widget class="QPushButton" name="EncryptPushButton">
   <property name="geometry">
    <rect>
     <x>80</x>
     <y>200</y>
     <width>171</width>
     <height>61</height>
    </rect>
   </property>
   <property name="text">
    <string>Encrypt</string>
   </property>
  </widget>
  <widget class="QLabel" name="PlainLabel">
   <property name="geometry">
    <rect>
     <x>100</x>
     <y>30</y>
     <width>91</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string>Plain (string)</string>
   </property>
  </widget>
  <widget class="QLabel" name="KeyLabel">
   <property name="geometry">
    <rect>
     <x>710</x>
     <y>30</y>
     <width>91</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string>key (integer)</string>
   </property>
  </widget>
  <widget class="QLabel" name="CipherLabel">
   <property name="geometry">
    <rect>
     <x>430</x>
     <y>30</y>
     <width>101</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string>Cipher (string)</string>
   </property>
  </widget>
  <widget class="QTextBrowser" name="OutputTextBrowser">
   <property name="geometry">
    <rect>
     <x>180</x>
     <y>330</y>
     <width>491</width>
     <height>121</height>
    </rect>
   </property>
  </widget>
  <widget class="QLineEdit" name="PlainLineEdit">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>60</y>
     <width>251</width>
     <height>71</height>
    </rect>
   </property>
  </widget>
  <widget class="QLineEdit" name="CipherLineEdit">
   <property name="geometry">
    <rect>
     <x>330</x>
     <y>60</y>
     <width>281</width>
     <height>71</height>
    </rect>
   </property>
  </widget>
  <widget class="QLineEdit" name="KeyLineEdit">
   <property name="geometry">
    <rect>
     <x>690</x>
     <y>60</y>
     <width>111</width>
     <height>71</height>
    </rect>
   </property>
  </widget>
  <widget class="QLabel" name="CipherLabel_2">
   <property name="geometry">
    <rect>
     <x>420</x>
     <y>300</y>
     <width>91</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string>Result</string>
   </property>
  </widget>
  <widget class="QPushButton" name="DecryptPushButton">
   <property name="geometry">
    <rect>
     <x>340</x>
     <y>200</y>
     <width>171</width>
     <height>61</height>
    </rect>
   </property>
   <property name="text">
    <string>Decrypt</string>
   </property>
  </widget>
  <widget class="QPushButton" name="AnalyzePushButton">
   <property name="geometry">
    <rect>
     <x>600</x>
     <y>200</y>
     <width>171</width>
     <height>61</height>
    </rect>
   </property>
   <property name="text">
    <string>Analyze</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections>
  <connection>
   <sender>buttonBox</sender>
   <signal>accepted()</signal>
   <receiver>QtCaesar</receiver>
   <slot>accept()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>248</x>
     <y>254</y>
    </hint>
    <hint type="destinationlabel">
     <x>157</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>buttonBox</sender>
   <signal>rejected()</signal>
   <receiver>QtCaesar</receiver>
   <slot>reject()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>316</x>
     <y>260</y>
    </hint>
    <hint type="destinationlabel">
     <x>286</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
